docs/services/vision/detection.md

```diff
@@ -318,13 +337,83 @@ if len(detectionsFromImage) > 0 {
 
 ```
 
+To learn more about how to use detection, see the [Go SDK docs](https://pkg.go.dev/go.viam.com/rdk/vision).
+
+    {{% /tab %}}
+    {{< /tabs >}}
+
+### Existing images
+
+If you would like to test your detector with existing images, load the images and pass them to the detector:
+
+{{< tabs >}}
+{{% tab name="Python" %}}
+
+```python {class="line-numbers linkable-line-numbers"}
+from viam.services.vision import VisionClient, VisModelConfig, VisModelType
+from PIL import Image
+
+robot = await connect()
+# Grab Viam's vision service for the detector
+my_detector = VisionClient.from_robot(robot, "my_detector")
+
+# Load an image
+img = Image.open('test-image.png')
+
+# Apply the detector to the image
+detections_from_image = await my_detector.get_detections(img)
+
+await robot.close()
+```
+
+To learn more about how to use detection, see the [Python SDK docs](https://python.viam.dev/autoapi/viam/services/vision/index.html).
+
+{{% /tab %}}
+{{% tab name="Go" %}}
+
+```go {class="line-numbers linkable-line-numbers"}
+import (
+  "go.viam.com/rdk/config"
+  "go.viam.com/rdk/services/vision"
+  "image/jpeg"
+  "os"
+)
+
+visService, err := vision.from_robot(robot=robot, name='my_detector')
+if err != nil {
+    logger.Fatalf("Cannot get Vision Service: %v", err)
+}
+
+// Read image from existing file
+file, err := os.Open("test-image.jpeg")
+if err != nil {
+    logger.Fatalf("Could not get image: %v", err)
+}
+defer file.Close()
+img, err := jpeg.Decode(file)
+if err != nil {
+    logger.Fatalf("Could not decode image: %v", err)
+}
+defer img.Close()
+
+// Apply the detector to the image
+detectionsFromImage, err := visService.Detections(context.Background(), img, nil)
+if err != nil {
+    logger.Fatalf("Could not get detections: %v", err)
+}
+if len(detectionsFromImage) > 0 {
+    logger.Info(detectionsFromImage[0])
+}
```

Comment:
I was not able to test this code - I get a session expiry error on all detections code currently

Reply:
nvm - tested and works